deployConfigs/dmmFactory.json

```diff
@@ -0,0 +1,17 @@
+{
+  "mainnet": "0x833e4083B7ae46CeA85695c4f7ed25CDAd8886dE",
+  "ropsten": "0x0639542a5cd99bd5f4e85f58cb1f61d8fbe32de9",
+  "polygon": "0x5F1fe642060B5B9658C15721Ea22E982643c095c",
+  "polygon_testnet": "0x7900309d0b1c8D3d665Ae40e712E8ba4FC4F5453",
+  "bsc_testnet": "0x7900309d0b1c8D3d665Ae40e712E8ba4FC4F5453",
+  "bsc": "0x878dFE971d44e9122048308301F540910Bbd934c",
+  "avax_testnet": "0x878dFE971d44e9122048308301F540910Bbd934c",
+  "avax": "0x10908C875D865C66f271F5d3949848971c9595C9",
+  "ftm_testnet": "0x10908C875D865C66f271F5d3949848971c9595C9",
+  "ftm": "0x78df70615ffc8066cc0887917f2Cd72092C86409",
+  "cronos": "0xD9bfE9979e9CA4b2fe84bA5d4Cf963bBcB376974",
+  "aurora": "0xD9bfE9979e9CA4b2fe84bA5d4Cf963bBcB376974",
+  "arbitrum": "0xD9bfE9979e9CA4b2fe84bA5d4Cf963bBcB376974",
+  "bttc": "0xD9bfE9979e9CA4b2fe84bA5d4Cf963bBcB376974",
+  "oasis": "0xD9bfE9979e9CA4b2fe84bA5d4Cf963bBcB376974"
```

Comment:
I think we also have Velas

Reply:
The factory could be different for different versions, like new Classic protocol should have the same Factory contract across all networks

Reply:
Will add different address for different versions according to engineering wiki docs

Reply:
btw, the address that is used across all networks(or almost networks) is defined by field "default" in the config files